test/E2ETests/SmokeTestsOnNanoServer.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using E2ETests.Common;
+using Microsoft.AspNetCore.Server.Testing;
+using Microsoft.AspNetCore.Testing.xunit;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace E2ETests
+{
+    public class SmokeTestsOnNanoServer : IDisposable
+    {
+        private readonly XunitLogger _logger;
+        private readonly RemoteDeploymentConfig _remoteDeploymentConfig;
+
+        public SmokeTestsOnNanoServer(ITestOutputHelper output)
+        {
+            _logger = new XunitLogger(output, LogLevel.Information);
+
+            var configuration = new ConfigurationBuilder()
+                .SetBasePath(Directory.GetCurrentDirectory())
+                .AddJsonFile("remoteDeploymentConfig.json")
```

Comment:
Don't you have to include this file in the `content` property of the `project.json`? Otherwise it doesn't get copied to output